.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Closes <!-- issue reference -->
+
+## Type of change
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes a bug)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Refactoring (only inner code structure was changed)
+- [ ] Documentation update (no code change was made)
```

Comment:
Are these explanations in parenthesis really necessary?
I'd either incorporate them into the text or drop them entirely.

Reply:
No, actually there are only for us and I think we know what their meaning is